### PR TITLE
refactor: routing & breadcrumb of account orders (history/detail)

### PR DESCRIPTION
**Pull request by Sebastian-Haehnlein**
## PR Type

 [x] Bugfix  
 [x] Feature  
 [ ] Code style update (formatting, local variables)  
 [x] Refactoring (no functional changes, no API changes)  
 [ ] Build-related changes  
 [ ] CI-related changes  
 [ ] Documentation content changes  
 [ ] Application / infrastructure changes  
 [ ] Other: 

## What Is the New Behavior?

Breadcrumb for order details display the order id.
Refactoring acount & orders routes.
(Issue Number: ISREST-461)

## Does this PR Introduce a Breaking Change?

 [ ] Yes  
 [x] No

### CODE REVIEW

**Comment by SGrueber** [comment]
if the order list is long (longer than the screen) the loading animation isn't displayed. This isn't most probably a new issue, pls fix it nevertheless.

File: src/app/core/store/orders/orders.effects.ts [hunk 1]
```diff
@@ -216,4 +231,24 @@ export class OrdersEffects {
...
+    switchMapTo(this.store.pipe(select(getSelectedOrder))),
+    whenTruthy(),
+    debounceTime(0),
+    withLatestFrom(this.translateService.get('account.orderdetails.breadcrumb')),
+    map(
+      ([order, x]) =>
+        new SetBreadcrumbData({
+          breadcrumbData: [
+            { key: 'account.order_history.link', link: '/account/orders' },
+            { text: `${x} - ${order.id}` },
```
Comment (SGrueber) [inline src/app/core/store/orders/orders.effects.ts]: the id should not be displayed, use the documentNo instead

File: src/app/core/store/orders/orders.effects.ts [hunk 2]
```diff
@@ -216,4 +231,24 @@ export class OrdersEffects {
...
   );
+
+  @Effect()
+  setOrderBreadcrumb$ = this.actions$.pipe(
+    ofRoute(),
+    mapToParam('orderId'),
+    whenTruthy(),
+    switchMapTo(this.store.pipe(select(getSelectedOrder))),
+    whenTruthy(),
+    debounceTime(0),
```
Comment (SGrueber) [inline src/app/core/store/orders/orders.effects.ts]: why is this needed? if it is needed it is missing in the quoteRequestBreadcrumb effect

File: src/app/core/store/orders/orders.effects.ts [hunk 3]
```diff
@@ -216,4 +231,24 @@ export class OrdersEffects {
     ofType(UserActionTypes.LogoutUser),
     mapTo(new ordersActions.ResetOrders())
   );
+
+  @Effect()
+  setOrderBreadcrumb$ = this.actions$.pipe(
```
Comment (SGrueber) [inline src/app/core/store/orders/orders.effects.ts]: test is missing

File: src/app/pages/account-order/account-order/account-order.component.html [hunk 4]
```diff
@@ -63,7 +63,7 @@ <h1>{{ 'account.orderdetails.heading.default' | translate }}</h1>
   ></ish-line-item-list>
 
   <!-- Cost Summary -->
-  <div class="section cost-summary float-right">
+  <div class="section cost-summary">
```
Comment (SGrueber) [inline src/app/pages/account-order/account-order/account-order.component.html]: the summary widget should not be displayed with the full width, because labels and numbers are too far away from each other. That's why this float-right class cannot be removed

File: src/app/core/store/orders/orders.effects.ts (hunk 2 above)
Comment (Sebastian-Haehnlein) [reply src/app/core/store/orders/orders.effects.ts]: the others are "features" and they are lazy loadet.
in this case we need it to be sure the translateservice works as expected

File: src/app/pages/account-order/account-order/account-order.component.html (hunk 4 above)
Comment (Sebastian-Haehnlein) [reply src/app/pages/account-order/account-order/account-order.component.html]: looks broken to me xD
